cmd/user-agent/src/pages/chapi/Login.vue

```diff
@@ -62,10 +62,10 @@ SPDX-License-Identifier: Apache-2.0
         },
         methods: {
             ...mapActions({loginUser: 'login', loadUser: 'loadUser', loadOIDCUser: 'loadOIDCUser'}),
-            ...mapGetters(['getCurrentUser', 'getAgentOpts']),
+            ...mapGetters(['getCurrentUser', 'getAgentOpts', 'serverURL']),
             ...mapGetters('agent', {getAgentInstance: 'getInstance'}),
             beginOIDCLogin: async function() {
-                window.location.href = "/oidc/login"
+                window.location.href = this.serverURL() + "/oidc/login"
```

Comment:
This shouldn't be necessary. The relative URL `/oidc/login` should suffice.

Hosting the frontend in a different domain from the backend will break things because the backend restricts cookies to its domain.

Reply:
@llorllale  I fixed cookies.

Reply:
@soluchok they didn't need fixing.

Reply:
It is necessary in case if we have the same domain but different ports.

Reply:
https://tools.ietf.org/html/rfc6265#section-8.5

Good finding @soluchok